src/compute/src/compute_state.rs

```diff
@@ -358,6 +359,11 @@ impl<'a, A: Allocate + 'static> ActiveComputeState<'a, A> {
 
         dyncfg_updates.apply(&self.compute_state.worker_config);
         dyncfg_updates.apply(&self.compute_state.persist_clients.cfg().configs);
+        self.compute_state
```

Comment:
Even if we can't require its use, I'd much prefer this logic to live in some `PersistConfig::apply` method

Reply:
Done. I called the method `PersistConfig::sync_from`, for symmetry with `configs_synced_once`. Let me know if you'd prefer `PersistConfig::apply_from` instead. (I think the `_from` suffix forms a nice symmetry with `clone` and `clone_from`.)

Reply:
I think minor preference for `apply_from` over `sync_from` (dyncfg seems like the bigger noun to me than configs_synced_once), but don't eat a CI cycle over it!

Reply:
Done! (CI was broken anyway.)